src/i18n/en.json

```
@@ -67,28 +69,26 @@
   "reports_title": "Doppler | Reports",
   "signup": {
     "activate_account_instructions": "* By clicking on the button from the Email, you will activate your account and you will be ready to enjoy all the benefits of Doppler.",
-    "button_signup": "Sign Up",
+    "button_signup": "create your free account",
     "check_inbox": "Check your inbox. You have an email!",
     "check_inbox_icon_description": "Check your inbox",
-    "copyright": "© {year} Doppler LLC. All rights reserved.",
+    "copyright": "© {year} Doppler LLC. All rights reserved. <a href=\"https://www.fromdoppler.com/en/legal/privacy-policy\">Privacy and Legal Policy.</a>",
```

Maybe:

```suggestion
    "copyright": "© {year} Doppler LLC. All rights reserved. <a target=\"_blanc\" href=\"https://www.fromdoppler.com/en/legal/privacy-policy\">Privacy and Legal Policy.</a>",
```

Take it into account in other links.

---

Ok I add it!